lib/components/ms-checkbox/MsCheckbox.vue

```diff
@@ -0,0 +1,59 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-checkbox
+    class="checkbox"
+    v-model="value"
+    @ion-change="onChange($event.detail.checked)"
+    :indeterminate="indeterminate"
+    aria-label=""
+  >
+    {{ $msTranslate(label) || '' }}
+  </ion-checkbox>
+</template>
+
+<script setup lang="ts">
+import { IonCheckbox } from '@ionic/vue';
+import { ref, watch, onUnmounted } from 'vue';
+
+const props = defineProps<{
+  modelValue?: boolean;
+  label?: string;
+  checked?: boolean;
+  indeterminate?: boolean;
+}>();
+
+const emits = defineEmits<{
+  (e: 'change', value: boolean): void;
+  (e: 'update:modelValue', value: boolean): void;
+}>();
+
+const watchModelCancel = watch(
+  () => props.modelValue,
+  (newValue: boolean) => {
+    value.value = newValue;
+  },
+);
+
+const watchCheckedCancel = watch(
+  () => props.checked,
+  (newValue: boolean) => {
+    value.value = newValue;
+  },
+);
+
+const value = ref(props.modelValue);
+
+onUnmounted(() => {
+  watchModelCancel();
+  watchCheckedCancel();
+});
+
+async function onChange(newValue: boolean): Promise<void> {
+  value.value = newValue;
+  emits('update:modelValue', newValue);
+  emits('change', newValue);
+}
+</script>
+
+<style lang="scss" scoped></style>
```

Comment:
Where is the CSS though?
Don't you think in this case it would be more logical to have it directly in the component instead of the theme global dist?